mpf/platforms/virtual.py

```diff
@@ -324,6 +324,10 @@ def set_text(self, text: str, flashing: FlashingType):
         self.text = text
         self.flashing = flashing
 
+    def set_color(self, colors: any) -> None:
```

Comment:
According to the base class this method should not be needed.

Reply:
I've updated the base class to be an abstract method and consolidated the ColorSegmentDisplay into the SegmentDisplay. Just did a check and now the virtual mode is happy with this declaration.